oracle-linux-image-tools/cloud/vagrant-virtualbox/provision.sh

```diff
@@ -0,0 +1,196 @@
+  chcon -u system_u -r object_r -t modules_conf_t /etc/modprobe.d/nofloppy.conf
+
+  # Customize the initramfs
+  # Enable VMware PVSCSI support for VMware Fusion guests.
+  echo 'add_drivers+=" mptspi "' > /etc/dracut.conf.d/vmware-fusion-drivers.conf
+  restorecon /etc/dracut.conf.d/vmware-fusion-drivers.conf
+  # There's no floppy controller, but probing for it generates timeouts
+  echo 'omit_drivers+=" floppy "' > /etc/dracut.conf.d/nofloppy.conf
+  restorecon /etc/dracut.conf.d/nofloppy.conf
+  # Regenerate initrd
+  local current_kernel=$(uname -r)
+  ${DRACUT_CMD} -f "/boot/initramfs-${current_kernel}.img" "${current_kernel}"
+
+  # Set SELinux to enforcing
+  sed -i -e 's/^SELINUX\s*=.*/SELINUX=enforcing/' /etc/selinux/config
+
+  # Install additional Packages
+  yum install -y ${YUM_VERBOSE} wget \
+    oracle-epel-release-el7 \
+    oraclelinux-developer-release-el7 \
+    oracle-softwarecollection-release-el7
+
+  # Enable optional repos
+  yum-config-manager --enable  ol7_addons >/dev/null
+  yum-config-manager --enable  ol7_preview >/dev/null
+  yum-config-manager --enable  ol7_developer >/dev/null
+  yum-config-manager --enable  ol7_developer_EPEL >/dev/null
+  yum-config-manager --enable  ol7_optional_latest >/dev/null
+
+  # Add login banner
+  echo "
+Welcome to Oracle Linux Server release `cat /etc/os-release | grep ^VERSION= | grep -o "[0-9].[0-9]"` (GNU/Linux $(uname -r))
+
+The Oracle Linux End-User License Agreement can be viewed here:
+
+  * /usr/share/eula/eula.en_US
+
+For additional packages, updates, documentation and community help, see:
+
+  * http://yum.oracle.com/
+  " > /etc/motd
+
+}
+
+#######################################
+# Install Virtualbox guest agent
+# Globals:
+#   YUM_VERBOSE
+# Arguments:
+#   None
+# Returns:
+#   None
+#######################################
+cloud::install_agent()
+{
+  echo_message "Install guest agent"
+  local additions="/mnt/VBoxLinuxAdditions.run"
+  yum install -y ${YUM_VERBOSE} make gcc bzip2 kernel-uek-devel
+
+  # Search for guest additions on cd devices
+  for cdrom in /dev/sr*; do
+    if mount -o ro "${cdrom}" /mnt; then
+      if [[ -f ${additions} ]]; then
+        # Found!
+        break
+      else
+        echo_message "No guest additions on ${cdrom}"
+      fi
+    else
+      echo_message "No media in ${cdrom}"
+    fi
+  done
+
+  [[ -f ${additions} ]] || echo_error "Guest additions not found"
+
+  sh "${additions}"
+  umount /mnt
+}
+
+#######################################
+# Provisioning module
+# Globals:
+#   None
+# Arguments:
+#   None
+# Returns:
+#   None
+#######################################
+cloud::provision()
+{
+  cloud::install_agent
+  cloud::config
+}
+
+#######################################
+# Cleanup module
+# Globals:
+#   None
+# Arguments:
+#   None
+# Returns:
+#   None
+#######################################
+cloud::cleanup()
+{
+  distr::remove_rpms usermode \
+    rhn-check.noarch \
```

Comment:
I suggest removing `rhn*` if that's possible. It'll remove all the yum packages too, via dependencies.

Reply:
Done in 1298637
(no impact on the selected packages, but cleaner)